src/legacy/status_im/ui/screens/signing/views.cljs

```diff
@@ -57,11 +54,7 @@
                      :style {:margin-right 8}}
                     display-symbol]
                    (if icon
-                     [wallet.components/token-icon
-                      (assoc icon
-                             :style       {:background-color colors/gray-lighter
-                                           :border-radius    16}
-                             :image-style {:width 24 :height 24})]
+                     [react/view]
```

Comment:
afaik signing screens can be removed later on too as they are just related to wallet? 🤔